Sources/Clappr/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -697,3 +697,26 @@ extension AVFoundationPlayback {
         return 5
     }
 }
+
+// MARK: - Metrics
+extension AVFoundationPlayback {
```

Comment:
Can we move this part to a file named `AVFoundationPlayback+Bitrate.swift`? That way we can avoid this "MARK" comment.

Reply:
Done ;-)